gax/src/main/java/com/google/api/gax/rpc/OperationCallSettings.java

```diff
@@ -74,6 +75,16 @@ private OperationCallSettings(
     this.metadataTransformer = metadataTransformer;
   }
 
+  @Override
+  public String toString() {
+    return MoreObjects.toStringHelper(this)
+        .add("initialCallSettings", getInitialCallSettings())
+        .add("pollingAlgorithm", getPollingAlgorithm())
```

Comment:
no reason to use getters in the same class, just reference the private fields directly.

Reply:
Fixed. 
I noticed, that `metadataTransformer` can be null, when other properties can't:
```
    this.initialCallSettings = checkNotNull(initialCallSettings);
    this.pollingAlgorithm = checkNotNull(pollingAlgorithm);
    this.responseTransformer = checkNotNull(responseTransformer);
    this.metadataTransformer = metadataTransformer;
```
is it intentionally or by mistake?